### PR TITLE
July 7 Release notes

### PR DESCRIPTION
**Pull request by gboone**
Six weeks worth of releases here!

### CODE REVIEW

File: content/updates/2017-07-07-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* [cf-service-connect]({{< relref "docs/services/relational-database.md#manually-access-a-database" >}}) now supports Windows. [See how to install it here.](https://github.com/18F/cf-service-connect#local-installation)
+
+### Changed
+
+- When you make a new service account or identity provider service instance, you will now [get the credentials using a service key]({{< relref "updates/2017-07-07-changes-to-credentials-broker.md" >}}), instead of getting the credentials from a Toaster (formerly Fugacious) link. This doesn’t change existing service account and identity provider instances, but you can delete and recreate them to use this new method.
+- [Static IP addresses for communicating with external apps]({{< relref "docs/apps/static-egress.md" >}}): Outbound traffic from cloud.gov now comes from specific IP addresses to help customers open up a connection between cloud.gov and outside data centers.
+- We recommend you upgrade your Cloud Foundry CLI to [the latest version](https://github.com/cloudfoundry/cli/releases) to get the latest bug fixes.
+- We improved the responsiveness of the dashboard to better reflect actions that happened on the backend.
```
Comment (jcscottiii) [inline content/updates/2017-07-07-release-notes.md:18]: i think it's `happened` -> `happen`

File: content/updates/2017-07-07-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - [Go v1.8.4](https://github.com/cloudfoundry/go-buildpack/releases/tag/v1.8.4)
+  - [NodeJS v 1.5.36](https://github.com/cloudfoundry/nodejs-buildpack/releases/tag/v1.5.36)
+  - [Python v1.5.19](https://github.com/cloudfoundry/python-buildpack/releases/tag/v1.5.19)
+  - [PHP v4.3.34](https://github.com/cloudfoundry/php-buildpack/releases/tag/v4.3.34)
+  - [Binary v1.0.13](https://github.com/cloudfoundry/binary-buildpack/releases/tag/v1.0.13)
+  - [.NET core v1.0.19](https://github.com/cloudfoundry/dotnet-core-buildpack/releases/tag/v1.0.19)
+
+### See Also
+
+If you’re interested in details about recent dashboard updates, you can also see the dashboard release notes.
```
Comment (jcscottiii) [inline content/updates/2017-07-07-release-notes.md:47]: Should we link to this page? https://github.com/18F/cg-dashboard/releases ? No strong opinions either way.

Comment (brittag) [reply content/updates/2017-07-07-release-notes.md:47]: Yeah! That's in [the doc version](https://docs.google.com/document/d/1Km3z2H8aH3wN-j2RguobNUhWdV7Hngu6AxwC6SHLbW8/edit#), guessing the link was just accidentally left out. :)

**Comment by brittag** [comment]
It looks like this build is failing because this branch is a bunch of commits behind master, so this branch doesn't have the other post from today, and Concourse is trying to help us prevent broken links.

So it looks like this needs to be rebased on master, or (easy shortcut version) make a new branch with the new post and make a new PR with that. :)

File: content/updates/2017-07-07-release-notes.md (hunk 2 above)
Comment (gboone) [reply content/updates/2017-07-07-release-notes.md:47]: Oops, yeah that's what happened.

**Comment by gboone** [comment]
OK, let's try this again. New PR is #957. Apparently I reused this branch name without deleting it first. Oops!

**Pull request closed without merging**